packages/cohort_banking/src/callbacks/oo_installer.rs

```diff
@@ -163,7 +165,7 @@ impl OutOfOrderInstallerImpl {
 
         // Quickly grab the snapshot to check whether safepoint condition is satisfied. Any row can be used for that.
         let (_, _, _, snapshot) = &result[0];
-        if (*snapshot as u64) < safepoint {
+        if &(*snapshot as u64) < safepoint {
```

Comment:
are we comparing reference? Why `&` ?

Reply:
aaah that was left over of something I was trying and forgot to clean up. Thanks for noticing this, I will update this.